src/devices/RGBDSensorClient/RGBDSensorClient.h

```diff
@@ -48,6 +48,8 @@ class RGBDSensor_StreamingMsgParser;
  * | remoteImagePort |      -         | string  | -              |   -           |               | Full name of the port to read color images from, e.g. /robotName/RGBD/image_camera:o  |       |
  * | remoteDepthPort |      -         | string  | -              |   -           |               | Full name of the port to read depth images from, e.g. /robotName/RGBD/depth_camera:o  |       |
  * | remoteRpcPort   |      -         | string  | -              |   -           |               | Full name of the remote RPC port, e.g. /robotName/RGBD/rpc                            |       |
+ * | ImageCarrier    |      -         | string  | -              | udp           | No            | Carrier name for the Image stream used in the client-server connection                |       |
+ * | DepthCarrier    |      -         | string  | -              | udp           | No            | Carrier name for the Depth strean used in the client-server connection                |       |
```

Comment:
Probably it is too late for reviewing it, but I'd prefer to have parameters lower-cases like the others.

cc @randaz81 @drdanz @traversaro